### PR TITLE
Add itest logging documentation

### PR DESCRIPTION
**Pull request by brendan-hofmann**
#### What does this PR do?

Document the itest log level flag.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?

@rzwiefel 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Build](https://github.com/orgs/codice/teams/build)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@lessarderic

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
Backport is up. https://github.com/codice/ddf/pull/1369

File: distribution/test/itests/README.md [hunk 1]
```diff
@@ -32,3 +32,6 @@ mvn clean test -DkeepRuntimeFolder=true
 ```
 
 The runtime folder used during the test will be available under `target/exam/<GUID>`.  It is possible to rerun the instance and verify that all bundles (excluding the test probe) are installed and working properly.  You can also inspect the logs under `target/exam/<GUID>/data/logs`.
+
+## Adjusting the Log Level
+By default, itests are run at a log level of `warn` for increased performance. If you want to change the logging level, use the flag `-DitestLoggingLevel=<level>`. Valid levels are defined by the [SLF4J API](http://www.slf4j.org/api/org/apache/commons/logging/Log.html).
```
Comment (lessarderic) [inline distribution/test/itests/README.md]: I believe the flag is `-DitestLogLevel`

**Comment by brendan-hofmann** [comment]
@lessarderic Fixed on both branches. 😅

File: distribution/test/itests/README.md [hunk 2]
```diff
@@ -32,3 +32,6 @@ mvn clean test -DkeepRuntimeFolder=true
 ```
 
 The runtime folder used during the test will be available under `target/exam/<GUID>`.  It is possible to rerun the instance and verify that all bundles (excluding the test probe) are installed and working properly.  You can also inspect the logs under `target/exam/<GUID>/data/logs`.
+
+## Adjusting the Log Level
+By default, itests are run at a log level of `warn` for increased performance. If you want to change the logging level, use the flag `-DitestLogLevel=<level>`. Valid levels are defined by the [SLF4J API](http://www.slf4j.org/api/org/apache/commons/logging/Log.html).
```
Comment (coyotesqrl) [inline distribution/test/itests/README.md]: Is that really necessary in the README inside the `itests` directory? Seems like the context should be enough to clarify that.

**Pull request merged**